app/models/review_bid.rb

```diff
@@ -21,29 +21,29 @@ def self.bidding_data(assignment_id, reviewer_ids)
   end
 
   # assigns topics to reviews as matched by the webservice algorithm
-  def self.assign_review_topics(assignment_id, reviewer_ids, matched_topics, _min_num_reviews = 2)
+  def assign_review_topics(assignment_id, reviewer_ids, matched_topics, _min_num_reviews = 2)
     # if review response map already created, delete it
     if ReviewResponseMap.where(reviewed_object_id: assignment_id)
       ReviewResponseMap.where(reviewed_object_id: assignment_id).destroy_all
     end
     # loop through reviewer_ids to assign reviews to each reviewer
     reviewer_ids.each do |reviewer_id|
-      topics_to_assign = matched_topics[reviewer_id.to_s]
+      topics_to_assign = matched_topics[reviewer_id]
       topics_to_assign.each do |topic|
         assign_topic_to_reviewer(assignment_id, reviewer_id, topic)
       end
     end
   end
 
   # method to assign a single topic to a reviewer
-  def self.assign_topic_to_reviewer(assignment_id, reviewer_id, topic)
+  def assign_topic_to_reviewer(assignment_id, reviewer_id, topic)
     team_to_review = SignedUpTeam.where(topic_id: topic).pluck(:team_id).first
     team_to_review.nil? ? [] : ReviewResponseMap.create(reviewed_object_id: assignment_id, reviewer_id: reviewer_id, reviewee_id: team_to_review, type: 'ReviewResponseMap')
   end
 
   # method for getting individual reviewer_ids bidding data
```

Comment:
What does this mean by "bidding data"?  The set of bids that has been placed by the reviewer?